lib/active_merchant/billing/gateways/nmi.rb

```diff
@@ -4,6 +4,7 @@ class NmiGateway < Gateway
       include Empty
 
       DUP_WINDOW_DEPRECATION_MESSAGE = 'The class-level duplicate_window variable is deprecated. Please use the :dup_seconds transaction option instead.'
+      LEVEL3_FIELDS = [:tax, :shipping, :ponumber]
```

Comment:
To be consistent with what exists in Active Merchant, I would prefer that we set the fields in the method call. AM leans toward explicitness and since the constant is used only once, it's a slight pre-mature optimization. 

@jasonxp That said, I don't want to discourage you from this type of improvement. I think it's great that you are thinking of simplification, reuse, and long-term maintainability.

Reply:
Sure. Since the fields are already in the ```add_level3_fields``` one-liner, it's already clear that they're level 3 fields, so I agree there isn't much value in the constant.